pluginkit/evaluation_suite.go

```diff
@@ -20,45 +20,59 @@ type EvaluationSuite struct {
 	StartTime string `yaml:"start-time"` // StartTime is the time the plugin started
 	EndTime   string `yaml:"end-time"`   // EndTime is the time the plugin ended
 
-	CorruptedState bool `yaml:"corrupted-state"` // BadState is true if any testSet failed to revert at the end of the evaluation
+	CorruptedState bool `yaml:"corrupted-state"` // CorruptedState is true if any testSet failed to revert at the end of the evaluation
 
 	EvaluationLog layer4.EvaluationLog `yaml:"control-evaluations"` // EvaluationLog is a slice of evaluations to be executed
 
-	//The plugin will pass us a list of the assessment requirements so that we can build our results, mainly used
-	//for populating the recommendation field.
-	requirements map[string]*layer2.AssessmentRequirement
+	catalog *layer2.Catalog // The Catalog this evaluation suite references
 
-	payload interface{}    // payload is the data to be evaluated
-	loader  DataLoader     // loader is the function to load the payload
-	config  *config.Config // config is the global configuration for the plugin
+	payload       interface{}    // payload is the data to be evaluated
+	loader        DataLoader     // loader is the function to load the payload
+	changeManager *ChangeManager // changes is a list of changes made during the evaluation
+	config        *config.Config // config is the global configuration for the plugin
 
 	evalSuccesses int // successes is the number of successful evaluations
 	evalFailures  int // failures is the number of failed evaluations
 	evalWarnings  int // warnings is the number of evaluations that need review
 }
 
+// AddChangeManager sets up the change manager for the evaluation suite
+func (e *EvaluationSuite) AddChangeManager(cm *ChangeManager) {
+	e.changeManager = cm
```

Comment:
Should this be inside the if statement? only set it if invasive? then you don't have to check for nil also. wdyt? 🤔